### PR TITLE
Avoid using the too heavy lodash

### PR DESCRIPTION
**Pull request by cbosse-skwirrel**

### CODE REVIEW

**Comment by jefferymills** [comment]
This is the only thing keeping from using this library right now. 84% of the size is lodash: https://bundlephobia.com/result?p=string-to-color@2.0.1

**Comment by Gustu** [comment]
> This is the only thing keeping from using this library right now. 84% of the size is lodash: https://bundlephobia.com/result?p=string-to-color@2.0.1

I'll check that

File: index.js [hunk 1]
```diff
@@ -15,11 +12,11 @@ module.exports = function(object) {
 };
 
 function getColors(text) {
-  var words = _words(text);
+  var words = text.match(/\w+/);
```
Comment (halcaponey) [inline index.js]:
```javascript
'test red'.match(/\w+/); // [ "test" ]
'test red'.match(/\w+/g); // [ "test", "red" ]
```
[lodash words](https://lodash.com/docs#words)

**Comment by Gustu** [comment]
It's not mergeable. Tests fail.

Current version is enough. Probably in 3.0.0, I'll remove color names, so all lodash functions may be removed.

**Pull request closed without merging**